temoa-uiuc/Snakefile

```diff
@@ -13,9 +13,9 @@ rule all:
         expand(r"data_files/{db}_{scenario}_model/{scenario}_OutputLog.log",
                zip,
                db=DBS, scenario=SCENARIOS),
-        expand(r"data_files/{db}_{scenario}_model/{scenario}.lp",
-               zip,
-               db=DBS, scenario=SCENARIOS),
+        # expand(r"data_files/{db}_{scenario}_model/{scenario}.lp",
+        #        zip,
+        #        db=DBS, scenario=SCENARIOS),
         expand("figures/{scenario}_{sector}_capacity.png", sector=SECTORS,
```

Comment:
does this require the pre-existence of the folder 'figures' inside temoa-uiuc/ or do those lines create the folder too?

Reply:
The folder is also created by the snakefile! The rule "all" tells snakemake what to look for in the outputs and if a required folder doesn't exist it is created.